pkg/chunk/disk_cache.go

```diff
@@ -112,7 +113,12 @@ func (cache *cacheStore) checkFreeSpace() {
 			cache.cleanup()
 			cache.Unlock()
 
-			br, fr := cache.curFreeRatio()
+			br, fr = cache.curFreeRatio()
+			if br < cache.freeRatio/2 || fr < cache.freeRatio/2 {
+				atomic.StoreInt32(&cache.pauseWB, 1)
```

Comment:
The address of cache.pauseWB should be aligned.

Since it's OK to see stale value of `pauseWB`, we don't need use atomic here.